extensions/2.0/Vendor/EXT_mesh_primitive_restart/schema/EXT_mesh_primitive_restart.schema.json

```
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "$id": "EXT_mesh_primitive_restart",
+  "title": "EXT_mesh_primitive_restart glTF extension",
```

```suggestion
  "title": "EXT_mesh_primitive_restart glTF Mesh Primitive Extension",
```

Also, this schema file should use 4-space indentation to match other schema files in this repo.

---

Thanks, addressed.